dependencies-path.yaml

```diff
@@ -0,0 +1,54 @@
+- dependency: "etcd"
+  refPaths:
+  - build/root/WORKSPACE
+  - cluster/gce/config-test.sh
+  - cluster/gce/manifests/etcd-empty-dir-cleanup.yaml
+  - cluster/gce/manifests/etcd.manifest
+  - cluster/gce/upgrade-aliases.sh
+  - cluster/images/etcd-empty-dir-cleanup/Makefile
+  - cluster/images/etcd/Makefile
+  - cluster/images/etcd/README.md
+  - cluster/images/etcd/migrate-if-needed.sh
+  - cmd/kubeadm/app/constants/constants.go
```

Comment:
etcd has client and server versions that can drift apart.

Reply:
I'm not sure we should track meta-dependencies

Reply:
agree to what @yastij said

Reply:
there has been a clean distinction between these two in release notes thus far.